packages/fx-core/templates/azure/config.bicep

```diff
@@ -0,0 +1,3 @@
+@secure()
```

Comment:
seems some test files are included in this PR (e.g. this file). Can you remove these files as well as update test cases that generate these files?

Reply:
sure, thanks